packages/terra-list/src/SingleSelectList.jsx

```diff
@@ -129,7 +136,11 @@ class SingleSelectList extends React.Component {
       newProps.onKeyDown = onKeyDown;
     }
 
+    // Uses the props.hasChevron value, unless the consumer specifies the item hasChevron attribute as false.
     newProps.hasChevron = this.props.hasChevrons;
+    if (item.props.hasChevron === false) {
```

Comment:
Ah I see what you are saying. Very good point, this does need to check for true or false. I was not thinking about it in the opposite way. Added an example & test.